monocle/db/queries.py

```diff
@@ -91,12 +91,25 @@ def generate_events_filter(params, qfilter):
 def generate_changes_filter(params, qfilter):
     state = params.get('state')
     tests_included = params.get('tests_included')
+    has_issue_tracker_links = params.get('has_issue_tracker_links')
     if state:
         qfilter.append({"term": {"state": state}})
     if tests_included:
         qfilter.append(
```

Comment:
should we do the same thing here with a "should" request?

Reply:
Why ? to match the test regexp in title or text ?

Reply:
no but to have boolean result instead of the files.

Reply:
Sorry I don't get it :(

Reply:
ok let's discuss it irl.

Reply:
understood